doc/release/index.rst

```diff
@@ -6,30 +6,105 @@
 Releases
 ========
 
-This section contains release notes for all Tarantool versions, starting from 1.6 up to the latest.
+This section contains information about Tarantool releases: release notes, lifecycle
+information, release policy, and other documents.
 To download Tarantool releases, check the `Download <https://www.tarantool.io/en/download/>`_ page.
 
-Since version 2.10, a :doc:`new release policy for Tarantool <policy>` has been introduced.
+All currently supported versions are listed on this page :ref:`below <release-supported-versions>`.
+The information about earlier versions is provided in :ref:`release-eol-versions`.
+
+The Enterprise Edition of Tarantool is distributed in the form of an SDK that has
+its own versioning. See the :ref:`release-enterprise-changelog` to learn about
+SDK version numbering and changes.
+
+The detailed information about Tarantool version numbering and release lifecycle
+is available in :ref:`release-policy`.
 
 Backwards compatibility is guaranteed between all versions in the same :term:`release series`.
 It is also appreciated but not guaranteed between different release series (major number changes).
 To learn more, read the :doc:`Compatibility guarantees <compatibility>` article.
 
-..  toctree::
-    :maxdepth: 2
+..  _release-supported-versions:
+
+Supported versions
+------------------
+
+Every Tarantool release series has :ref:`the same lifecycle <release-series-lifecycle>`
+defined by the release policy. The following diagram visualises the lifecycle of currently
+supported Tarantool versions:
+
+.. ifconfig:: builder not in ('latex', )
+
+    .. image:: _images/releases_calendar.svg
+        :target: ../../_images/releases_calendar.svg
+        :align: center
+        :alt: releases_calendar.svg
+
+The table below provides information about supported versions with links to their
+*What's new* pages in the documentation and detailed changelogs on GitHub.
+For information about earlier versions, see :doc:`eol_versions`.
+
+.. note::
+    *End of life* (*EOL*) means the release series will no longer receive any patches,
+    updates, or feature improvements after the specified date. Versions that haven't
+    reached their end of life yet are shown in **bold**.
+
+    *End of support* (*EOS*) means that we won't provide technical support to product
+    versions after the specified date.
 
-    calendar
-    major-features
+..  container:: table
+
+    ..  list-table::
+
+        *   -   Series
+            -   First release date
+            -   End of life
+            -   End of support
+            -   Versions
+
+        *   -   :doc:`3.0 </release/3.0.0>`
+            -   **December 26, 2023**
```

Comment:
I'd remove bold formatting from regular table rows - looks like header rows sneaked into the middle of the table.  IMO, additional marking of such rows in bold is excessive and adds some visual clutter. 

And I'd also add _header_ formatting to the first row (Series, First release date, ...).